SnapWar/ViewControllers/SWAuthViewController.swift

```diff
@@ -39,15 +39,13 @@ class SWAuthViewController: UIViewController, UITextFieldDelegate {
         passwordField.tag = 1
         formContainer.addArrangedSubview(passwordField)
         
-        loginButton.setTitle("Login", for: .normal)
-        loginButton.backgroundColor = SWColorType.primary
         loginButton.addTarget(self, action: #selector(handleSubmit), for: .touchUpInside)
-        setLoginButtonState()
+        loginButton.isEnabled = isFormValid()
```

Comment:
hmmm i feel like you should wrap this line inside a method

Reply:
oki